fe/loss.py

```diff
@@ -21,7 +21,7 @@ def mybar_impl(w):
     return A
 
 
-def mybar_vjp(g, w):
+def mybar_jvp(g, w):
```

Comment:
This was previously discussed starting [here](https://github.com/proteneer/timemachine/pull/539#discussion_r754718532).